cmd/kube-apiserver/app/aggregator.go

```diff
@@ -61,6 +61,13 @@ func createAggregatorConfig(kubeAPIServerConfig genericapiserver.Config, command
 	etcdOptions.StorageConfig.Codec = aggregatorapiserver.Codecs.LegacyCodec(v1beta1.SchemeGroupVersion)
 	genericConfig.RESTOptionsGetter = &genericoptions.SimpleRestOptionsFactory{Options: etcdOptions}
 
+	if err := commandOptions.APIEnablement.ApplyTo(
```

Comment:
add a comment: // override MergedResourceConfig with aggregator defaults and registry